### PR TITLE
Implement the large font for accessibility.

### PR DESCRIPTION
**Pull request by wanchao-xu**
- The system setting provide enum value for font size (SYSTEM_SETTINGS_FONT_SIZE_SMALL, SYSTEM_SETTINGS_FONT_SIZE_NORMAL, SYSTEM_SETTINGS_FONT_SIZE_LARGE, SYSTEM_SETTINGS_FONT_SIZE_HUGE and SYSTEM_SETTINGS_FONT_SIZE_GIANT). 

```
#define MIDDLE_FONT_DPI                   (-100)
#define SMALL_FONT_DPI                     (-80)
#define LARGE_FONT_DPI                      (-150)
#define HUGE_FONT_DPI                       (-190)
#define GIANT_FONT_DPI                      (-250)
```

### CODE REVIEW

**Comment by swift-kim** [comment]
@WonyoungChoi Do you have any idea about the windows gclient sync failure? It keeps failing for some reason.

**Comment by WonyoungChoi** [comment]
@swift-kim It's strange..  I'm trying to reproduce the error on my local environment. 

### Error
```
________ running 'python3 src/flutter/tools/download_dart_sdk.py --fail-loudly' in 'C:\workspace\engine'
Traceback (most recent call last):

  File "src/flutter/tools/download_dart_sdk.py", line 293, in <module>

    sys.exit(Main())

  File "src/flutter/tools/download_dart_sdk.py", line 242, in Main

    version = utils.ReadVersionFile()

AttributeError: module 'utils' has no attribute 'ReadVersionFile'
```

File: shell/platform/tizen/channels/settings_channel.cc [hunk 1]
```diff
@@ -59,4 +67,36 @@ bool SettingsChannel::Prefer24HourTime() {
...
+        break;
+      case SYSTEM_SETTINGS_FONT_SIZE_GIANT:
+        scale = giant;
+        break;
+      default:
+        break;
+    }
+    return scale;
+  }
+  return 1.0;
```
Comment (swift-kim) [inline shell/platform/tizen/channels/settings_channel.cc:99]: How about

```cpp
int value = SYSTEM_SETTINGS_FONT_SIZE_NORMAL;
if (system_settings_get_value_int(SYSTEM_SETTINGS_KEY_FONT_SIZE, &value) ==
    SYSTEM_SETTINGS_ERROR_NONE) {
  switch (value) {
    case SYSTEM_SETTINGS_FONT_SIZE_SMALL:
      return small;
    case SYSTEM_SETTINGS_FONT_SIZE_LARGE:
      return large;
    case SYSTEM_SETTINGS_FONT_SIZE_HUGE:
      return huge;
    case SYSTEM_SETTINGS_FONT_SIZE_GIANT:
      return giant;
    default:
      return normal;
  }
}
return normal;
```

File: shell/platform/tizen/channels/settings_channel.cc [hunk 2]
```diff
@@ -59,4 +67,36 @@ bool SettingsChannel::Prefer24HourTime() {
   return false;
 }
 
+float SettingsChannel::GetTextScaleFactor() {
+  const float small = 0.8;
+  const float normal = 1.0;
+  const float large = 1.5;
+  const float huge = 1.9;
+  const float giant = 2.5;
```
[the remark this answers is not in the document]
Comment (wanchao-xu) [reply shell/platform/tizen/channels/settings_channel.cc:75]: Not based on experiment, I asked the member of sysytem setting, these numbers is defined in framework.
```
#define MIDDLE_FONT_DPI                   (-100)
#define SMALL_FONT_DPI                     (-80)
#define LARGE_FONT_DPI                      (-150)
#define HUGE_FONT_DPI                       (-190)
#define GIANT_FONT_DPI                      (-250)
```

File: shell/platform/tizen/channels/settings_channel.cc (hunk 1 above)
Comment (wanchao-xu) [reply shell/platform/tizen/channels/settings_channel.cc]: I fixed it.

**Review by swift-kim — APPROVED** [review]
I tested this change on mobile/wearable and it worked very well. Good job.

(But I personally wish there is a medium font size between "normal" and "large" because 1.5 feels too large, especially on wearable.)

**Comment by WonyoungChoi** [comment]
@wanchao-xu Could you rebase this PR? to solve the CI errors.

**Comment by swift-kim** [comment]
Will this feature be supported on TV devices as well? (e.g. by enabling Focus Magnification in the accessibility settings)

**Comment by wanchao-xu** [comment]
> Will this feature be supported on TV devices as well? (e.g. by enabling Focus Magnification in the accessibility settings)

This feature isn't supported on TV. TV can't get the value of `SYSTEM_SETTINGS_KEY_FONT_SIZE`.
The accessibility settings has `enlarge` mode to magnify all widget, not only font. and `enlarge` is implemened by `Tizen.TV.FLUX.NUI`.

**Pull request merged**